packages/e2e-tests/scripts/save-examples.ts

```diff
@@ -194,9 +194,13 @@ async function saveExamples(
     const allExampleNames = examplesToRun.map(e => e.filename);
     const invalidInputs = difference(specificExamples, allExampleNames);
     if (invalidInputs.length) {
-      throw new Error(
-        `Invalid examples don't exist or require manual recording: ${invalidInputs.join(",")}`
-      );
+      if (argv.target !== "all") {
+        // If no target is specified, this script will check both Chrome and Node examples;
+        // we shouldn't throw in that case (since it's likely that at least one target will contain no matches)
+        throw new Error(
+          `Invalid examples don't exist or require manual recording: ${invalidInputs.join(",")}`
+        );
+      }
```

Comment:
This got broken somewhere along the way. Not sure when. Lots of folks were tweaking this file.

Reply:
I remember adding the `if (invalidInputs.length) { ... }` part - Not sure I ever saw the `argv.target` part 👀